fala/apps/adviser/templatetags/adviser_extras.py

```diff
@@ -66,6 +66,15 @@ def category_selection(form):
     return None
 
 
+@library.filter
+def category_selection_first_item(form):
+    if "categories" in form.cleaned_data:
+        categories = [PROVIDER_CATEGORIES[cat] for cat in form.cleaned_data["categories"]]
+        if categories:
+            return categories[0]
+    return None
```

Comment:
```
@library.filter
def category_selection_first_item(form):
    """Returns the first category from cleaned_data['categories'], if available."""
    categories = form.cleaned_data.get("categories", [])

    if categories:
        return PROVIDER_CATEGORIES.get(categories[0], None)

    return None
```
What do you think? 

There were a lot of checks in this code. Avoids the unnecessary "categories" in form.cleaned_data check.
.get("categories", []) returns an empty list by default.

PROVIDER_CATEGORIES[cat] could raise a KeyError.
Uses `.get()` to return None if the key is missing.

Reply:
Happy to tweak to that.